src/wizard-settings/header-bidding-gam/index.js

```diff
@@ -224,56 +124,125 @@ const HeaderBiddingGAM = () => {
 							error.message
 						) : (
 							<span className="newspack-ads__header-bidding-gam__order-description">
-								{ order?.order_id ? (
-									<span>
-										{ __( 'Order:', 'newspack-ads' ) }{ ' ' }
-										<a
-											href={ getOrderUrl( order.order_id ) }
-											target="_blank"
-											rel="noopener noreferrer"
-										>
-											{ order.order_name }
-										</a>
-										{ ', ' }
-										{ order?.line_item_ids
-											? sprintf(
-													// Translators: Number of line items in the order.
-													_n(
-														'containing %s line item',
-														'containing %s line items',
-														order.line_item_ids.length,
-														'newspack-ads'
-													),
-													order.line_item_ids.length
-											  )
-											: __( 'missing line items configuration' ) }
-										.
-									</span>
-								) : (
-									getMissingOrderMessage()
-								) }
+								{ activeOrders.length
+									? sprintf(
+											// Translators: Number of line items in the order.
+											_n(
+												'There is %s available order.',
+												'There are %s available orders.',
+												activeOrders.length,
+												'newspack-ads'
+											),
+											activeOrders.length
+									  )
+									: getMissingOrderMessage() }
 							</span>
 						) }
 					</Fragment>
 				) }
-				checkbox={ isValid() ? 'checked' : 'unchecked' }
-				actionText={ canConfigure() ? __( 'Configure', 'newspack-ads' ) : null }
-				onClick={ () => setIsCreating( true ) }
+				actionText={ canConfigure() ? cardActionText : null }
+				onClick={ () => ( activeOrders.length ? setIsManaging( true ) : setEditingOrder( 0 ) ) }
 			/>
-			{ isCreating && (
+			{ isManaging && editingOrder === false && (
 				<Modal
-					title={ __( 'Create Order', 'newspack-ads' ) }
-					onRequestClose={ () => ! inFlight && setIsCreating( false ) }
+					title={ __( 'Manage Orders', 'newspack-ads' ) }
+					onRequestClose={ () => ! inFlight && setIsManaging( false ) }
 				>
-					<p>
-						{ __(
-							'Create the order and line items on your Google Ad Manager network according to the pre-defined price bucket settings.',
-							'newspack-ads'
-						) }
-					</p>
-					{ error && error.data?.status !== '404' && (
-						<Notice isError noticeText={ error.message } />
+					{ activeOrders.length && (
+						<>
+							<Card noBorder>
+								{ activeOrders.map( order => (
+									<ActionCard
+										key={ order.id }
+										title={ order.name }
```

Comment:
Non-blocking: can this title be a link to the edit view?

Reply:
It'd be nice, but `<ActionCard />` doesn't support `titleLink` as anything other than a `href` string. We don't expect users to edit orders too often, so I think it's fine.